### DISCUSSION TITLE
EasyWeChat 服务端能不能回复文章？

### DISCUSSION THREAD

**Discussion by DaveCherish** [w7corp/easywechat#2391] [Q&A]
![JX13P( 0$ O%ZL_VPHLVH45](https://user-images.githubusercontent.com/59271647/157214097-15c5d3cb-c07f-4b6b-a2f3-9c4caedecc6c.png)

![ENC} YF$1YYJU 9~SWP1T69](https://user-images.githubusercontent.com/59271647/157214126-306f59b1-f85a-438d-a00f-6e418298e9db.png)

**Comment by overtrue** [comment]
更新你的版本
